### PR TITLE
31030 table styling

### PR DESCRIPTION
**Pull request by bienvu**
Please review !

### CODE REVIEW

**Comment by JohnAlbin** [comment]
1. Table border should be 2px wide, not 1px.

2. Cell padding should be `11px 22px`.

File: themes/agov/agov_whitlam/sass/base/tables/tables-table.hbs [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<table>
-  <caption>Optional table caption.</caption>
+<table class="table">
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/base/tables/tables-table.hbs]: We don't need a `table` class added. This is the default table element styling.

**Comment by bienvu** [comment]
Thanks for feedback.
I fixed it.
Please review again!

**Comment by JohnAlbin** [comment]
Hmm… the table headings are not very bold. I think we need to add `font-family: $font-heading;` to make the `th` elements use Open Sans Bold. Or maybe create a new Sass variable? I'll leave that to you to decide.

**Comment by bienvu** [comment]
I'm merge code from branch 7.x-3.x and update font for table headings.
Please check!

File: themes/agov/agov_whitlam/sass/base/tables/_tables.scss [hunk 2]
```diff
@@ -24,5 +24,12 @@ table {
 
 td,
 th {
-  padding: 0;
+  text-align: left;
+  border: 2px solid color(border-color);
+  padding: 11px 22px;
+}
+
+tr th {
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/base/tables/_tables.scss]: Does the selector need to be `tr th`? Wouldn't `th` suffice?

**Pull request merged**